intro-checkpoint.ipynv

```diff
@@ -0,0 +1,172 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "74c98243",
+   "metadata": {},
+   "source": [
+    "# Write a program to create \"intro.txt\" file.  Print only even number of row."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47c567ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "id": "c96c2ef4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=open('intro.txt','w',newline=\"\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "id": "750c2f7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " k=csv.writer(a)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "id": "10e397eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l=[['my name is ansh'],['i am 18 year old'],['I live in chalthan'],\n",
+    "   ['my name is ayuh'],[ 'I am live in bardoli'],['i am in collage'],\n",
+    "  ['i am bca student'],['i am belong to up']]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "id": "60ff849c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[['my name is ansh'], ['i am 18 year old'], ['I live in chalthan'], ['my name is ayuh'], ['I am live in bardoli'], ['i am in collage'], ['i am bca student'], ['i am belong to up']]\n"
+     ]
+    }
+   ],
+   "source": [
+    "k.writerows(l)\n",
+    "print(l)"
```

Comment:
k.writerows(l) means?

Reply:
it will write the multiple rows which are stored in variable 'l'